api_tests/base/test_throttling.py

```diff
@@ -0,0 +1,77 @@
+import mock
+
+from nose.tools import *  # flake8: noqa
+
+from api.base.settings.defaults import API_BASE
+
+from tests.base import ApiTestCase
+from tests.factories import AuthUserFactory, ProjectFactory
+
+
+class TestUserRateThrottle(ApiTestCase):
+
+    def setUp(self):
+        super(TestUserRateThrottle, self).setUp()
+        self.user = AuthUserFactory()
+        self.url = '/{}'.format(API_BASE)
+
+    @mock.patch('rest_framework.throttling.UserRateThrottle.allow_request')
+    def test_user_rate_allow_request_called(self, mock_allow):
+        res = self.app.get(self.url, auth=self.user.auth)
+        assert_equal(res.status_code, 200)
+        assert_not_equal(len(mock_allow.mock_calls), 0)
```

Comment:
Two things:

1. In general, it's better to be specific with the number of calls, because knowing that something was called too many times is at least as good as knowing it wasn't called at all. Having the mocks properly configured to be able to catch multiple calls to any throttling would have caught the problem with the "too many defaults", which leads me to…
2. If we can catch something that happens for all of the throttle calls (base_throttle.get_ident maybe?) as well, then you could be sure that you're only getting requests for two different throttles on default routes and maybe three on others.